Leyrin-Aguilar/tarea-1/style.css

```diff
@@ -0,0 +1,65 @@
+
+*{
```

Comment:
I see several code format issues, try to use an extension for VSCode like Prettier